Update add-admin-contacts.md

mertenar opened the pull request:
@jaimeo changes reflect new UI on MMD admin portal

mertenar committed:
Update add-admin-contacts.md

@jaimeo changes reflect new UI on MMD admin portal

jaimeo commented:
The "areas of focus" are concepts, right? That is, not formally named entities. If so, then do not use title case (i.e., "IT help desk" and "App packaging").

mypil commented:
@mertenar - kindly check on the response from jaimeo and apply the suggested changes.

Thank you.

mertenar committed:
Update add-admin-contacts.md

edited per Jaime's suggestion - let's await reply from Stuart (Jaime see thread 8/5) before committing

mertenar commented:
edited per @jaimeo 's suggestion - let's await reply from Stuart (Jaime see thread 8/5) before committing
